wordpress.org/public_html/wp-content/plugins/plugin-directory/email/class-plugin-approved.php

```diff
@@ -29,8 +29,8 @@ function body() {
 To answer some common questions:
 
 * You must use SVN to upload your code -- we are unable to do that for you
-* Your SVN username is %2$s and your password is the same as you use to log in to WordPress.org
-* Your username is case sensitive
+* Your SVN username is %2$s and is case sensitive.
+* Generate your SVN password in your <a href="https://profiles.wordpress.org/%2$s/profile/edit/group/3/?screen=svn-password">WordPress.org profile</a>.
```

Comment:
It appears that this might be a plain-text email, and not a HTML one.

This can probably be converted to a Markdown-based email
 - Change it to extend `Markdown_Base` instead of `Base`
 - Rename `function body()` to `function markdown()`

Test.

To test the email, you can do something like this:
```
( new WordPressdotorg\Plugin_Directory\Email\Plugin_Approved( get_post( 12345 ), get_user_by( 'slug', 'steve...' ) ) )->send();

or
$email = new WordPressdotorg\Plugin_Directory\Email\Plugin_Approved( get_post( 12345 ), get_user_by( 'slug', 'steve...' ) );

echo $email->html();

```

Reply:
Thanks for that. That does seem to work.